wire-library/wire-schema/src/commonMain/kotlin/com/squareup/wire/schema/Linker.kt

```diff
@@ -274,11 +274,16 @@ class Linker {
   }
 
   /** Returns the current package name from the context stack. */
-  fun packageName(): String? {
+  fun fullyQualifiedNameOfContext(): String {
```

Comment:
Oooooh I don’t like this function name as much! Might be necessary if it’s more accurate though. And this isn’t really facing so it’s fine!

Reply:
I'm happy to accept proposals for other names. Maybe `packageOrEnclosingTypeName`?

Reply:
Yeah that's much better. Let's use that.

Reply:
So... I think we can actually just _remove_ this method and use the existing `resolveContext` (but make it non-private). It looks like it already does the right thing.

Also, I was thinking of changing this in a few places where it's named `packageName` to instead be called `namespace` and adding doc comments to clarify that it's either a package name or the name of the enclosing message. For this I am referring to the eponymous field on the `Field` type, and parameters of the same name in `Extend.fromElements`, `Field.fromElements`, and `OneOf.fromElements`.